cmd/opm/serve/serve.go

```diff
@@ -103,9 +122,70 @@ func (s *serve) run(ctx context.Context) error {
 	health.RegisterHealthServer(grpcServer, server.NewHealthServer())
 	reflection.Register(grpcServer)
 	s.logger.Info("serving registry")
+	p.stopStartupCpuProfiling()
```

Comment:
It's a question of scope.  We want to stop profiling CPU info for our custom endpoint once the GRPC server is ready.  If we defer in this scope, then it will stop profiling on process exit, which won't help. 
The intent with the custom endpoint is to profile CPU usage between the process startup and GRPC server readiness.